app/src/main/java/com/blockeq/stellarwallet/helpers/StellarRecoveryString.kt

```diff
@@ -21,7 +21,7 @@ class StellarRecoveryString (string : String, val isRecoveryPhrase : Boolean, va
         }
 
         if (isRecoveryPhrase) {
-            if ((wordCount != 12 && wordCount != 24) || invalidRecovery) {
+            if ((wordCount != 12 && wordCount != 15 && wordCount != 18 && wordCount != 21 && wordCount != 24) || invalidRecovery) {
```

Comment:
could you put a comment here with the link of the standard `https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki`?